src/data/history.ts

```diff
@@ -55,7 +55,8 @@ export const fetchRecent = (
   entityId,
   startTime,
   endTime,
-  skipInitialState = false
+  skipInitialState = false,
+  significantChangesOnly = true
```

Comment:
It's dangerous that we're encoding a set of defaults here on top of the defaults in the backend. Too late now for `skipIniitalState`, but let's not introduce another one with our own defaults.
```suggestion
  skipInitialState = false,
  significantChangesOnly?: boolean
```